gensim/models/keyedvectors.py

```diff
@@ -1692,8 +1690,6 @@ def most_similar(self, positive=None, negative=None, topn=10, clip_start=0, clip
     def doesnt_match(self, docs):
         """Which document from the given list doesn't go with the others from the training set?
 
-        TODO: Accept vectors of out-of-training-set docs, as if from inference.
```

Comment:
Docs still can't be `np.ndarray`, need to implement this too + add tests (same for others, exclude `most_similar`)